python/ray/serve/_private/deploy_utils.py

```diff
@@ -97,6 +97,7 @@ def deploy_args_to_deployment_info(
     route_prefix: Optional[str],
     is_driver_deployment: Optional[bool] = False,
     app_name: Optional[str] = None,
+    **kwargs,
```

Comment:
why do we need it? (not being used in this func)

Reply:
There are some args passed to this function from application state manager that isn't actually used by the function. Instead of having to prune the args to perfectly match the function, I added this here since it's cleaner.